src/main/java/nl/goodbytes/xmpp/xep0363/Component.java

```diff
@@ -114,6 +120,33 @@ protected IQ handleIQGet( IQ iq ) throws Exception
     {
         final Element request = iq.getChildElement();
         final Collection<String> namespaces = Arrays.asList( NAMESPACE, NAMESPACE_EXP );
+        // Implements the TYPE_IQ jabber:iq:version protocol (version info xep-0092). Allows
+        // XMPP entities to query each other's application versions.  The server
+        // will respond with its current version info.
+        if ("query".equals(request.getQName().getName()) && "jabber:iq:version".equals( request.getNamespaceURI())) {
+            try {
+                Element answerElement = DocumentHelper.createElement(QName.get("query", "jabber:iq:version"));
+                MavenXpp3Reader reader = new MavenXpp3Reader();
+                Model model = reader.read(new FileReader("pom.xml"));
+                answerElement.addElement("name").setText(model.getName());
+                answerElement.addElement("description").setText(model.getDescription());
+                answerElement.addElement("version").setText(model.getVersion());
+                answerElement.addElement("Organization").setText(model.getOrganization().getName());
```

Comment:
"Organization" is not a field defined in the XEP, so we'd better leave it off. Also, if the POM does not define an organization, I expect that `model.getOrganization().getName()` to result in a NullPointerException. It is good to explicitly check for these things.